CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# How To Contribute
+
+We'd love to accept your patches and contributions to this project. There are just a few guidelines you need to follow which are described in detail below.
+
+## 1. Fork this repo
+
+You should create a fork of this project in your account and work from there. You can create a fork by clicking the fork button in GitHub.
+
+## 2. One feature, one branch
+
+Work for each new feature/issue should occur in its own branch. To create a new branch from the command line:
+```shell
+git checkout -b my-new-feature
+```
+where "my-new-feature" describes what you're working on.
+
+## 3. Add unit tests
+
+If your contribution modifies existing or adds new code please add corresponding unit tests for this.
+
+## 4. Ensure that the build passes
+
+Run
+```shell
+mvn package
```

Comment:
This isn't a mvn project so you should either remove this step or replace it with a Terraform equivalent.

Reply:
will exchange for `terraform plan`